src/test/java/uk/gov/pay/directdebit/payments/api/PaymentViewValidatorTest.java

```diff
@@ -16,32 +22,109 @@
     private PaymentViewValidator validator = new PaymentViewValidator();
 
     @Test
-    public void shouldReturnNoErrors() {
-        PaymentViewSearchParams searchParams = new PaymentViewSearchParams("a-gateway-account", 3l, 256l);
+    public void shouldReturnNoErrorsWithMinimumParams() {
+        PaymentViewSearchParams searchParams = new PaymentViewSearchParams("a-gateway-account", 
+                3l, 256l, null, null);
```

Comment:
Very minor one: usually for long values it's good to use capital `L`, otherwise it looks like as number `1`